README.md

```diff
@@ -365,6 +362,23 @@ instance for a Mongo.Collection instance. For example:
 MyCollection.simpleSchema().validate(doc);
 ```
 
+## Schema Clean Options
+
+You can set clean options globally in collection2. 
```

Comment:
Maybe like this: "You can set the `simpl-schema` clean options globally in collection2. They are merged with any options defined on the schema level."